iri/apis/machine/v1alpha1/api.proto

```diff
@@ -99,6 +107,24 @@ message MachineSpec {
   repeated NetworkInterface network_interfaces = 6;
 }
 
+message Event {
+  meta.v1alpha1.ObjectMetadata metadata = 1;
+  EventSpec spec = 2;
+}
+
+message EventSpec {
+  string reason = 1;
+  string message = 2;
+  string type = 3;
+  int64 event_time = 4;
+  EventSeries event_series = 5;
```

Comment:
I think we can remove this information for now since it's calculated by the `EventRecorder` in `controller-runtime@/pkg/internal/recorder/recorder.go`

Reply:
Done, removed EventSeries